libs/SalesforceAnalytics/SalesforceAnalytics.xcodeproj/project.pbxproj

```diff
@@ -848,6 +848,7 @@
 				MTL_ENABLE_DEBUG_INFO = YES;
 				ONLY_ACTIVE_ARCH = YES;
 				SDKROOT = iphoneos;
+				SIGNPOST_ENABLED = 1;
```

Comment:
Why do we have it in the xcconfig and in the project file?

Reply:
I don't think you need that one if you have it in the GCC_PREPROCESSOR_DEFINITIONS through the xcconfig file.

Reply:
Agreed - let's just keep it in the `xcconfig` file if possible.